sdk/identity/azure-identity/tests/managed-identity-live/service-fabric/service_fabric.md

```diff
@@ -0,0 +1,190 @@
+
+To use the provided template:
+
+1. Open `arm-templates/cluster.parameters.json` and complete the fields `clusterLocation`, `adminUserName`, `adminPassword`, `sourceVaultValue`, `certificateUrlValue`, and `certificateThumbprint`. Field descriptions will describe how they should be completed.
+2. In `arm-templates/cluster.parameters.json`, change all instances of `sfmi-test` to a unique name, like `<myusername>-sfmi-test`. Also, change the values of `applicationDiagnosticsStorageAccountName` and `supportLogStorageAccountName` to be similarly unique, but without hyphens. This will help ensure the deployment resource names do not conflict with the names of other public resources.
+3. Start the deployment by running from your Powershell window in the `arm-templates` directory:
+```powershell
+Connect-AzAccount
+Select-AzSubscription -Subscription $Subscription
+New-AzResourceGroupDeployment -TemplateParameterFile ".\cluster.parameters.json" -TemplateFile ".\cluster.template.json" -ResourceGroupName $ResourceGroupName
+```
+
+This will begin deployment of a Service Fabric cluster, as well as other necessary resources: a load balancer, public IP address, virtual machine scale set, storage account, and virtual network.
+
+## Set Up and Deploy the Applications
+
+### Build and publish a Docker image for each application
+
+For this manual test, each application will use a Docker image to run managed identity tests. To make these images available to Service Fabric, you need to build and publish them by using the Dockerfiles in this sample.
+
+First, you'll need to update the applications to target the correct resources.
+
+1. Open `sfmitestsystem/Dockerfile`.
+2. Replace `<your vault URL>` with the vault URI of your key vault.
+```Dockerfile
+ENV AZURE_IDENTITY_TEST_VAULT_URL=<your vault URL>  # looks like https://<vault name>.vault.azure.net/
+```
+3. Open `sfmitestuser/Dockerfile`.
+4. Replace `<your vault URL>` with the vault URI of your key vault, and replace `<AdminUser client ID>` with the client ID of your managed identity.
+```Dockerfile
+ENV AZURE_IDENTITY_TEST_VAULT_URL=<your vault URL>  # looks like https://<vault name>.vault.azure.net/
+ENV AZURE_IDENTITY_TEST_MANAGED_IDENTITY_CLIENT_ID=<AdminUser client ID>  # found in the managed identity overview
+```
+
+Now, build the images and push them to Azure Container Registry.
+
+1. Ensure Docker is running and is using Linux containers.
+2. Authenticate to ACR:
+```
+az acr login -n $ACR_NAME
+```
+3. Build the images:
+```
+docker build --no-cache -t $ACR_NAME.azurecr.io/sfmitestsystem sfmitestsystem
```

Comment:
This gets the default value of PYTHON_VERSION, i.e. 2.7. We also need to test on 3.x (3.5 seems a good choice because it's our lowest supported version). You can set that by adding `--build-arg PYTHON_VERSION=3.5`.